fvm/context.go

```diff
@@ -110,10 +121,18 @@ func WithChain(chain flow.Chain) Option {
 	}
 }
 
-// WithGasLimit sets the gas limit for a virtual machine context.
-func WithGasLimit(limit uint64) Option {
```

Comment:
Thank you for making the naming consistent 👍

Reply:
I just noticed dkg test package is dependent on the emulator and the emulator is dependent on this method, added this method as well as a deprecated method, until we update the emulator and other places, we could remove it in the follow up PRs.